unit-tests/py/rspy/devices.py

```diff
@@ -496,8 +500,9 @@ def enable_only( serial_numbers, recycle = False, timeout = 5 ):
             #
             log.d( 'recycling ports via acroname:', ports )
             #
-            acroname.disable_ports( acroname.ports() )
-            _wait_until_removed( serial_numbers, timeout = timeout )
+            enabled_devices = enabled()
+            acroname.disable_ports(acroname.ports(), sleep_on_change=0)
```

Comment:
sleep_on_change 0 is the default... remove?

Reply:
Will remove

Reply:
Done